src/test-utils/dom/steps/index.ts

```diff
@@ -0,0 +1,39 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import { ComponentWrapper, ElementWrapper } from '@cloudscape-design/test-utils-core/dom';
+
+import statusIconStyles from '../../../status-indicator/styles.selectors.js';
+import testUtilStyles from '../../../steps/test-classes/styles.selectors.js';
+
+export class StepWrapper extends ComponentWrapper {
+  /**
+   * Finds the status of a step
+   */
+  findStatusIndicator(): ElementWrapper | null {
+    return this.findByClassName(statusIconStyles.icon);
+  }
```

Comment:
I'm not sure how helpful this would be given it only is an icon

Reply:
Isn't it useful so that consumers can test that they expected status is there?

Reply:
Not really, this would only return an icon element which is alone not enough to communicate the status, besides it won't really indicate the status unless they depend on the class name which is not recommended. If you check status indicator, it also doesn't have a selector for the icon.